swagger/sales/coupons.json

```diff
@@ -0,0 +1,96 @@
+{
+  "host": "api.vchost.co",
+  "swagger": "2.0",
+  "info": {
+    "title": "API V3",
+    "version": "v3.0"
+  },
+  "paths": {
+    "/v3/sales/coupons": {
+      "get": {
+        "summary": "Get coupons api",
+        "description": "Get list of coupons",
```

Comment:
you need to add a description including which token are supported (see examples in other APIs